paper.tex

```diff
@@ -95,7 +95,7 @@ \subsubsection{Broker}
 
 \subsection{Component}
 \subsubsection{IOTA}
-IOTA\cite{IOTAwhitepaper} is a cryptocurrency which the ledger does not consist of transactions grouped into blocks and stored in sequential chains, but a directed acyclic graph(DAG) of individual transactions, \textbf{Tangle}. To issue a transaction in the network, one needs to perform a small amount of Proof-of-Work (PoW) that verifies two previous transactions. Since every actor in Tangle has the same role, there is no need to offer transaction fees. Therefore, it is possible to make \textbf{micropayments} which the importance will increase in the rapidly developing IoT industry, and \textbf{zero-transactions} that one could store information securely within Tangle transactions. Moreover, this structure also enables high scalability of transactions that could resolve the drawbacks of blockchain and the concern of rapid growth of IoT devices.
+\textbf{IOTA}\cite{IOTAwhitepaper} is a cryptocurrency for the IoT industry based on a revolutionary distributed ledger technology, the \textbf{Tangle}, which the ledger consists of a directed acyclic graph(DAG) of individual transactions. To issue a transaction, one needs to perform a small amount of Proof-of-Work (PoW) that verifies two previous transactions. Since every actor in Tangle has the same role, IOTA is feeless with high scalability . Moreover, it is possible to make \textbf{zero-transactions} that one could store information securely within Tangle transactions.
```

Comment:
I would like to mention MAM here. IOTA should be referred within the paragraph about MAM. The concept of second layer data communication protocol such as MAM is important for our research. However, IOTA is not vital from the perspective of Interchangeability. Thus, let's merge the descriptions about MAM and IOTA into single component part.

Reply:
How about the order of each component? Do we need to reorder them?

Reply:
I'll reorder to:
* MAM
* TangleID
* Ethereum smart contract
* blind signature
* IPFS

Reply:
The proposed order looks good to me.